### DISCUSSION TITLE
[Bug] In springboot+rabbitmq projects, rabbitmq listener did not generates traceId automatically.

### DISCUSSION THREAD

**Discussion by hpsocket** [apache/skywalking#11301] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

In springboot+rabbitmq projects, rabbitmq listener did not generates traceId automatically.

The next codes print `TraceId: N/A` at the first line.

```
@RabbitListener(queues = {AppConfig.QUE_REGION_0}, ackMode = "MANUAL", containerFactory = "defaultSimpleRabbitListenerContainerFactory")
public void onDefaultMessage(Message message, Channel channel) throws IOException
{
	// print traceId
	System.out.println("TraceId: " + TraceContext.traceId());
		
        MessageProperties properties = message.getMessageProperties();

        String msgId		= properties.getMessageId();
        long deliveryTag	= properties.getDeliveryTag();
        String domainName	= properties.getHeader(HEADER_DOMAIN_NAME);
        String eventName	= properties.getHeader(HEADER_EVENT_NAME);
        
        try
        {
            JSONObject msg = JSON.parseObject(message.getBody());

            log.info("receive message -> msgId: {}, domainName: {}, evnetName: {}, msg: {}", msgId, domainName, eventName, msg.toJSONString());
            channel.basicAck(deliveryTag, false);
        }
        catch(Exception e)
        {
        	channel.basicNack(deliveryTag, false, false);
        	log.error("receive message fail -> msgId: {}, domainName: {}, evnetName: {}, exception: {}", msgId, domainName, eventName, e.getMessage(), e);
	}
}
```

How can slove this program ? And if it can  not generates traceId automatically, how can I generate traceId manually ?

### What you expected to happen

Generates traceId automatically at every time  invoke mq listener entry method.

### How to reproduce

Just create a rabbitmq listener, and consume any message.

### Anything else

![image](https://github.com/apache/skywalking/assets/142854599/303f1d6d-5d0e-490f-bf5d-66a15a8074f7)

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by githubcheng2978** [comment]
Because of the quoted thread, MQ receives the message is put to a queue inside the queue, the consumption of this queue created in spring。
so I can fix this bug ,pls assign to me

**Reply by wu-sheng** [reply]
This is a discussion, you could submit a pull request with this reference.
